lib/model/content.dart

```diff
@@ -1013,13 +1024,24 @@ class _ZulipContentParser {
 
   List<BlockContentNode> parseBlockContentList(dom.NodeList nodes) {
```

Comment:
So the other call site of `parseBlockContent` is above, in `parseImplicitParagraphBlockContentList`. What happens if images appear in a context where that gets called? I.e., at the top level of an element of a list.